content/docs/user-guide/assets/asset-processor/asset-cache-server.md

```diff
@@ -0,0 +1,105 @@
+
+# Setting Up Asset Processor in Asset Cache Server Mode
+
+The **Asset Cache Server** (ACS) mode has three steps to set up the system so that developers can use it.
+
+{{< note >}}
+The next sections assume the root folder to be `T:/o3de` but it can be in any folder or it can run on Linux.
+{{< /note >}}
+
+## Create the Transfer Folder
+
+The team will need to create a shared folder that all the team members will be able to access over the network. The tech should work between Windows and Linux, but the cacheServerAddress string only accepts a single string such as `T:/o3de/Transfer/Cache` that all the clients are meant to read from. It is possible to be clever about having an **Asset Processor** in ACS mode write to one configured `cacheServerAddress` string via Windows and having another ACS retrieve from a Samba link on Linux; but that will be a challenge the team will need to solve.
+
+## Run the Asset Processor in ACS in Server Mode
+
+The design of the ACS mode is to have one machine contributing to the asset cache in the remote folder and other **Asset Processor** clients running retrieve the cached product asset archive files.
+
+To enable the **Asset Processor** in ACS mode as a server, a .setreg file needs these settings:
+```
+{
+  "Amazon": {
+    "AssetProcessor": {
+      "Settings": {
+        "Server": {
+          "enableCacheServer": true,
+          "cacheServerAddress": "T:/o3de/cache_server",
+          "ACS FBX Glob": {
+            "glob": "*.fbx",
+            "checkServer":  true
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+This will enable the cache server mode for an Asset Processor where it will write out the cached archive files to `T:/o3de/cache_folder` for all FBX files.
+
+The setting key `/AssetProcessor/Settings/Server/enableCacheServer=true` enables the **Asset Processor** to run in "server mode". In this mode, the **Asset Processor** will detect source asset changes, process the product assets, archive the product files (in a ZIP archive), and save out the archive files in the remote folder.
+
+The setting key `/AssetProcessor/Settings/Server/cacheServerAddress=<remote_shared_path>` points to a remote folder that the server and all clients can read and write from, respectively. The transfer folder should already be set up before launching the **Asset Processor**.
+
+The setting key `/AssetProcessor/Settings/Server/ACS FBX Glob={}` object specifies FBX source assets as a file type to cache. There can be a number of entries specified in the settings registry where the entry’s title needs to start with the letters ACS such as "ACS Our Textures" and "ACS Audio Files". The "glob" pattern can be used to capture files by extension or some basic matching pattern. It is important to flag the entry with "checkServer" to true to enable the entry for caching.
```

Comment:
Minor style suggestions:

```suggestion
The previous example enables ACS mode for Asset Processor so that it writes the cached archive files to `T:/o3de/cache_folder` for all FBX files.

The setting key `/AssetProcessor/Settings/Server/enableCacheServer=true` enables Asset Processor to run in server mode. In server mode, Asset Processor detects source asset changes, processes the product assets, archives them (in a ZIP archive), and saves out the archive files in the remote directory.

The setting key `/AssetProcessor/Settings/Server/cacheServerAddress=<remote_shared_path>` points to a remote directory that the server and all clients can read from and write to. The transfer directory should be set up before launching Asset Processor.

The setting key `/AssetProcessor/Settings/Server/ACS FBX Glob={}` object specifies FBX source assets as a file type to cache. There can be a number of entries specified in the settings registry where the entry’s title needs to start with the letters ACS such as "ACS Our Textures" and "ACS Audio Files". The "glob" pattern can be used to capture files by extension or some basic matching pattern. It's important to flag the entry with `"checkServer": true` to enable the entry for caching.
```